data_pipelines_cli/__init__.py

```diff
@@ -5,4 +5,4 @@
 pipelines.
 """
 
-version = "0.25.3"
+version = "0.26.0"
```

Comment:
I think it's redundant here - version will be bumped in the PR into main created by the github actions bot.